### PR TITLE
Update getting-started

### PR DESCRIPTION
**Pull request by BerkeSinanYetkin**
Update getting-started to be inline with upstream examples.

as per #49.

### CODE REVIEW

**Comment by BerkeSinanYetkin** [comment]
I will mark it as ready for review once i figure out how to deal with these old commits

File: getting-started/robot.py [hunk 1]
```diff
@@ -1,46 +1,67 @@
...
+#
 
 import wpilib
 import wpilib.drive
 
+"""The VM is configured to automatically run this class, and to call the functions corresponding to
+each mode, as described in the TimedRobot documentation. If you change the name of this class or
+the package after creating this project, you must also update the manifest file in the resource
+directory.
+"""
```
Comment (auscompgeek) [inline getting-started/robot.py:15]: This is neither a docstring nor is it correct. Please remove this.

File: getting-started/robot.py [hunk 2]
```diff
@@ -13,34 +15,47 @@ def robotInit(self):
...
+        self.timer.restart()
 
     def autonomousPeriodic(self):
         """This function is called periodically during autonomous."""
 
         # Drive for two seconds
         if self.timer.get() < 2.0:
-            self.drive.arcadeDrive(-0.5, 0)  # Drive forwards at half speed
+            # Drive forwards half speed, make sure to turn input squaring off
+            self.robotDrive.arcadeDrive(0.5, 0, False)
```
Comment (auscompgeek) [inline getting-started/robot.py:41]: Small nit: could you please use a keyword argument to turn input squaring off?

**Review by auscompgeek — APPROVED** [review]
Looks good, just one last thing.

**Comment by auscompgeek** [comment]
Thank you!

**Pull request merged**